lib/charms/mysql/v0/mysql.py

```diff
@@ -175,6 +175,10 @@ class MySQLGetMySQLVersionError(Error):
     """Exception raised when there is an issue getting the MySQL version."""
 
 
+class MysqlGetClusterPrimaryAddressError(Error):
```

Comment:
minor nit: MysqlGetClusterPrimaryAddressError -> MySQLGetClusterPrimaryAddressError

Reply:
actually, have to do before on the lib!